addon/components/bs-modal.js

```diff
@@ -506,45 +508,33 @@ export default class Modal extends Component {
    * @param callback
    * @private
    */
-  handleBackdrop(callback) {
-    let doAnimate = this.usesTransition;
+  async handleBackdrop() {
```

Comment:
Would love to refactor `handleBackdrop` into two separate methods: `showBackdrop` and `hideBackdrop`. But this would also require renaming the existing `showBackdrop` property. And I wanted to avoid touching too much in one pull request. I may do that one later.